amundsen_application/static/js/ducks/lastIndexed/api/v0.ts

```diff
@@ -0,0 +1,15 @@
+import axios, { AxiosResponse, AxiosError } from 'axios';
+
+export const API_PATH = '/api/metadata/v0';
+
+type MessageAPI = { msg: string };
+
+export type LastIndexedAPI = { timestamp: string } & MessageAPI;
+
+export function getLastIndexed() {
+  return axios
+    .get(`${API_PATH}/get_last_indexed`)
+    .then((response: AxiosResponse<LastIndexedAPI>) => {
+      return response.data.timestamp;
+    });
```

Comment:
I would add a catch clause here and pass down the statusCode and maybe the errorMessage>

You can see an example in the dashboard code: https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/dashboard/api/v0.ts#L25

Reply:
Added catch phrase. But not included error code and message since currently we are not using error code and message if we do not put error message in Footer if we do not get last_indexed value.